config/brakeman.ignore

```diff
@@ -76,18 +83,21 @@
       },
       "user_input": "params[:list].presence",
       "confidence": "High",
+      "cwe_id": [
+        77
+      ],
       "note": "The params[:list] argument is already checked before the template gets rendered by @presenter.acceptable_list?(@list)."
     },
     {
       "warning_type": "SQL Injection",
       "warning_code": 0,
-      "fingerprint": "d3d42c84d0b0acf57ccb1c9d1133ca43389fd725d0471016013e52bf59269b83",
+      "fingerprint": "814bc0a997725602aa2913dd902c8d77c28de9044d4562f60a898ad250d04b5a",
       "check_name": "SQL",
       "message": "Possible SQL injection",
       "file": "lib/csv_report_generator.rb",
       "line": 11,
       "link": "https://brakemanscanner.org/docs/warning_types/sql_injection/",
-      "code": "Redis.current.lock(\"publisher:#{Rails.env}:report_generation_lock\", :life => 15.minutes)",
+      "code": "Redis.new.lock(\"publisher:#{Rails.env}:report_generation_lock\", :life => 15.minutes)",
```

Comment:
Shall we change the code so these no longer produce warnings. 

I can't see there being any need for us to have Rails.env inside these locking keys - it's hard to think of a scenario where you'd need that distinction. So we could just simplify them to get rid of needing to update this further.

Reply:
Would you object to my making this a new card on our backlog? There are some similar/identical brakeman warnings in other apps, and it's not really connected to the intention of this PR, so might make sense to complete the Redis work and then tackle the Brakeman work en masse after.

Reply:
@kevindew just trying to clear up some of Ed's cards while he's off - are you okay with his suggestion (make this change a new one in our backlog separate from the redis fix)?

Reply:
I see Kev's away, I'm going to say we can expedite this and catch the other work Ed mentions in this card: https://trello.com/c/84yWQ0NJ/1662-clean-up-brakeman-errors-following-on-from-rediscurrent-redisnew-updates

Reply:
Yep – that sounds good to me @KludgeKML 👍🏻